openapi3.yaml

```diff
@@ -7,6 +7,31 @@ info:
     name: MIT
     url: https://opensource.org/licenses/MIT
 paths:
+  /storage:
+    get:
+      tags:
+        - storage
+      summary: Get free, in-use and total storage size for exporting (in bytes)
```

Comment:
its written here 3 parameters that should be returned, but according to response scheme and the code api it seems that only 2 params always returned (no in use params)

Reply:
only free and size are returned

Reply:
Removed `in-use`.